lib/Inline/Python.pm6

```diff
@@ -542,7 +542,12 @@ BEGIN {
     PythonObject.^compose;
 }
 
-multi sub EVAL(Cool $code, Str :$lang where { ($lang // '') eq 'Python' }, PseudoStash :$context) is export {
-    state $py = ::("Inline::Python").default_python;
-    $py.run($code);
+multi sub EVAL(
+        Str $code,
```

Comment:
Is the change from Cool to Str intentional? This will make EVAL(1, :lang<Python>) fail for example. Not that it's terribly useful, but the other EVAL variants support it, too.